timeline/templates/timeline/_search_results.html

```diff
@@ -0,0 +1,75 @@
+{% load rules %}
+{% load timeline_tags %}
+
+{# Projectroles dependency #}
+{% load projectroles_common_tags %}
+
+<style type="text/css">
+  .table#sodar-tl-search-table thead tr th:nth-child(2) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table tbody tr td:nth-child(2) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table thead tr th:nth-child(4) {
+    width: 75px !important;
+  }
+
+  .table#sodar-tl-search-table tbody tr td:nth-child(4) {
+    width: 75px !important;
+  }
+
+  /* Responsive modifications */
+  @media screen and (max-width: 1300px) {
```

Comment:
These CSS media modifications could use a bit of tweaking for the search result list, but it's a bit hard to describe. I will do it when I clean up the v0.12.0 release :)